outputs.tf

```diff
@@ -1,8 +1,17 @@
 ########################################################################################################################
 # Outputs
 ########################################################################################################################
+output "isv_instance_name" {
+  description = "The name of the IBM Security Verify instance."
+  value       = ibm_resource_instance.isv_instance.name
+}
 
-#output "myoutput" {
-#  description = "Description of my output"
-#  value       = "value"
-#}
+output "crn" {
+  description = "The IBM Security Verify instance CRN."
+  value       = ibm_resource_instance.isv_instance.crn
+}
+
+output "guid" {
+  description = "The globally unique identifier of the IBM Security Verify instance."
+  value       = ibm_resource_instance.isv_instance.guid
+}
```

Comment:
perhaps output the dashboard url? It seems to be in this format `https://<hostname>.verify.ibm.com/ui/admin/` according to the [docs](https://www.ibm.com/docs/en/security-verify?topic=overview-accessing-security-verify)

Reply:
Not sure about what will be attribute name which will be referred after this ibm_resource_instance.isv_instance.<attribute_name>. My guess is it would be be account_url because in mails it is referred as Account URL.

<img width="1282" alt="Screenshot 2024-09-17 at 7 13 23 PM" src="https://github.com/user-attachments/assets/0f9f9221-0093-472f-ab48-ed69f869e7d2">

Reply:
@dishankkalra23 the `ibm_resource_instance` wont support outputting the url. And since there is no other ISV provider resources yet, I was suggesting to construct the output outselves. According to the doc, it in the format of `https://<hostname>.verify.ibm.com/ui/admin/`